bindings/cs/rl.net.native/rl.net.live_model.cc

```diff
@@ -15,7 +16,7 @@ static void pipe_background_error_callback(const reinforcement_learning::api_sta
     }
 }
 
-API livemodel_context_t* CreateLiveModel(reinforcement_learning::utility::configuration* config)
+API livemodel_context_t* CreateLiveModel(reinforcement_learning::utility::configuration* config, factory_context_t* factory_context)
```

Comment:
Yes, this is correct, but marking it `const` would not really do anything here, since this is an internal API intended to be used by bindings. Still, a fair point.